multi-agent/adapters/outbound/mongo/admin_config_reader.py

```
@@ -0,0 +1,105 @@
+        mongodb_port: MongoDB port.
+        db_name: Database that holds the ``admin_config`` collection
+                 (defaults to ``"config"`` — the backend's database).
+        coll_name: Collection name (defaults to ``"admin_config"``).
+        cache_ttl_seconds: How long a fetched admin list is reused across
+                 requests/processes before re-reading Mongo. Every
+                 admin-gated request otherwise triggers a read; a short TTL
+                 keeps that off the hot path without making admin-list
+                 changes take unreasonably long to propagate.
+    """
+
+    def __init__(
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)